scripts/collect-assets-workflow.js

```diff
@@ -0,0 +1,45 @@
+#! /usr/bin/env node
+import { ensureDir } from "fs-extra/esm";
+import { resolve, join } from "path";
+import { cwdToProjectRoot } from "./_util.js";
+import { downloadWorkflowArtifacts, downloadAndExtractTarGz } from "./_download.js";
+import dotenv from "dotenv";
+dotenv.config();
```

Comment:
do we read GH token from the `.env` file?

Reply:
yes. we need GH token for downloading artifacts from a pipeline run

Reply:
removed .env file. 

Using `export` to set env variables should just work